Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM public.ecr.aws/ubuntu/ubuntu@sha256:722b3bddfe96b95441f626cf94974d79213090ecbd16954f71d7c080fb413561
+
+LABEL org.opencontainers.image.vendor="Ministry of Justice" \
+      org.opencontainers.image.authors="Analytical Platform (analytical-platform@digital.justice.gov.uk)"
+      # uncomment if required to your repository details
+      #org.opencontainers.image.title="{image title}" \
+      #org.opencontainers.image.description="{decription}" \
+      #org.opencontainers.image.url="https://github.com/ministryofjustice/analytical-platform"
+
+ENV CONTAINER_USER="analyticalplatform" \
+    CONTAINER_UID="1000" \
+    CONTAINER_GROUP="analyticalplatform" \
+    CONTAINER_GID="1000" \
+    DEBIAN_FRONTEND="noninteractive"
+
+# User
+RUN groupadd \
+      --gid ${CONTAINER_GID} \
+      ${CONTAINER_GROUP} \
+    && useradd \
+         --uid ${CONTAINER_UID} \
+         --gid ${CONTAINER_GROUP} \
+         --create-home \
+         --shell /bin/bash \
+         ${CONTAINER_USER}
+
+# Base
+RUN apt-get update --yes \
+    && apt-get install --yes \
+         "apt-transport-https=2.4.11" \
+         "curl=7.81.0-1ubuntu1.15" \
+         "git=1:2.34.1-1ubuntu1.10" \
+         "gpg=2.2.27-3ubuntu2.1" \
+         "python3.10=3.10.12-1~22.04.3" \
+         "python3-pip=22.0.2+dfsg-1ubuntu0.4" \
+         "unzip=6.0-26ubuntu3.2" \
+    && apt-get clean --yes \
+    && rm --force --recursive /var/lib/apt/lists/*
+
+USER ${CONTAINER_USER}
+
+WORKDIR /home/${CONTAINER_USER}
+
+# Basic health check that always returns a success status
+HEALTHCHECK CMD true
+
+# Place build logic here:
+# Example
+# COPY . /home/${CONTAINER_USER}
+# RUN python -m pip install --no-cache-dir --upgrade pip==23.3.1 \
+#     && python -m pip install --no-cache-dir --requirement requirements.txt
+
+# CMD ["foo.handler"]
```

Comment:
I'm not sure if this is a good thing to include, they aren't always needed depending on what type of container you're writing

Reply:
removed added a healthcheck skip as checkov fails